js/packages/openinference-semantic-conventions/src/trace/SemanticConventions.ts

```diff
@@ -93,12 +93,20 @@ export const OUTPUT_MIME_TYPE =
   `${SemanticAttributePrefixes.output}.mime_type` as const;
 /**
  * The messages sent to the LLM for completions
- * Typically seen in openAI chat completions
+ * Typically seen in OpenAI chat completions
  * @see https://beta.openai.com/docs/api-reference/completions/create
  */
 export const LLM_INPUT_MESSAGES =
   `${SemanticAttributePrefixes.llm}.${LLMAttributePostfixes.input_messages}` as const;
 
+/**
+ * The prompts sent to the LLM for completions
+ * Typically seen in OpenAI legacy completions
+ * @see https://beta.openai.com/docs/api-reference/completions/create
+ */
+export const LLM_PROMPTS =
+  `${SemanticAttributePrefixes.llm}.${LLMAttributePostfixes.prompts}` as const;
```

Comment:
since you are changing this package you need to add a changeset using `pnpx changeset` and commit it

Reply:
addressed in fc6fb3d